internal/tutils/psql.go

```diff
@@ -0,0 +1,136 @@
+package tutils
+
+import (
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/osbuild/image-builder/internal/db"
+)
+
+type PSQLContainer struct {
+	name string
+	id   string
+	port int
+}
+
+const (
+	image string = "postgres:13"
```

Comment:
`podman` prefers fully-qualified names, so I suggest making this `docker.io/library/postgres:13`.